fractal.config.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const mandelbrot = require('@frctl/mandelbrot');
+const nunjucks = require('@frctl/nunjucks');
+
+const type = require('#/type');
+
+// Create a new Fractal instance and export it for use elsewhere if required
+const fractal = (module.exports = require('@frctl/fractal').create());
+
+// if we want to make our own reusable theme
+// const oddTheme = require('oddTheme');
+
+// Set the title of the project
+fractal.set('project.title', 'OddPieces');
+
+// Tell Fractal where the components will live
+fractal.components.set('path', `${__dirname}/content/_includes`);
+fractal.components.set('default.preview', '@preview');
+
+// Tell Fractal where the documentation pages will live
+fractal.docs.set('path', `${__dirname}/content/docs`);
+
+// Register the Nunjucks adapter for components
+const njk = nunjucks({
+  env: {
+    // Nunjucks environment opts: https://mozilla.github.io/nunjucks/api.html#configure
+  },
+  filters: {
+    // filter-name: function filterFunc(){}
+    mdInline: type.mdInline,
+  },
+  globals: {
+    // global-name: global-val
+  },
+  extensions: {
+    // extension-name: function extensionFunc(){}
+  },
+});
```

Comment:
I added one filter just to get the `title` component to render, but if we move forward we'd want to add all the filters currently defined in `.eleventy.js`. We also might want to import a global data set that could be available to all components, matching Eleventy's data structure?